### ISSUE TITLE
Implement SASS or LESS for handling CSS

### ISSUE THREAD

**Issue by thestrugglingblack** [blkCodeCollctve/bcc#29]
This is for fun but eventually sometime down the road we would like to utilize these css extensions to handle how we develop the design and look of our page.

[SASS DOCUMENTATION](http://sass-lang.com/documentation/file.SASS_REFERENCE.html)
[LESS DOCUMENTATION](http://lesscss.org/)

Acceptance Criteria:
- Either extension tools are being used within our CSS to condense repetition and readability .

**Comment by jmdembe** [comment]
I think I'll go with Sass on this one. We're using webpack, correct?

**Comment by thestrugglingblack** [comment]
Yes, we are using webpack.  In the next few days there is going to be some major updates done to webpack to resolve retrieving local files and getting over the CORS error.

**Comment by pdrittenhouse** [comment]
If using SASS & Webpack be sure to use the ruby-sass-loader as sass-loader uses node-sass/libsass and has issues with compiling nested placeholders/variables.
